db/migrate/20230309233731_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :first_name
+      t.string :email
+      t.string :last_name
+      t.date :birthday
+      t.string :country
+      t.string :password
```

Comment:
Just make sure to encrypt this in the controller

Reply:
okay!